src/loader/bgw_launcher.c

```diff
@@ -422,6 +447,8 @@ message_start_action(HTAB *db_htab, BgwMessage *message, VirtualTransactionId vx
 			bgw_message_send_ack(message, ACK_FAILURE);
 			return;
 		}
+		else
```

Comment:
NON-NIT: we should set the hash entry values for the `if(!found)` here too

Reply:
good point, done